battleship/board.py

```diff
@@ -1,20 +1,28 @@
 DIRECTIONS = ('NS', 'EW')
 OCEAN = 'O'
-SHIPS = [
-    {'type':'carrier','size': 5, 'code': 'C'},
-    {'type': 'battleship', 'size': 4, 'code': 'B'},
-    {'type': 'cruiser', 'size': 3, 'code': 'R'},
-    {'type': 'submarine', 'size': 3, 'code': 'U'},
-    {'type': 'destroyer', 'size': 2, 'code': 'D'},
-]
+
 
 class Board(object):
     def __init__(self):
         self.board = [[OCEAN for _ in range(10)] for _ in range(10)]
+        self.hit_ships = []
+        self.attacks = []
```

Comment:
I don't think we need to hold this information. Instead, let's make each attack site an M for Miss or an H for Hit. That way a player can print their current board at any point in the game.